### ISSUE TITLE
Check this test.css: Gutter Color vs Color Highlighter

### ISSUE THREAD

**Issue by LOQUILLO** [ggordan/GutterColor#69]
This is a screenshot with Gutter Color enabled (Color Highlighter disabled):
![test css_gutter-color](https://cloud.githubusercontent.com/assets/1308190/4005480/82125f60-2992-11e4-8263-336991a77bd3.png)

This is a screenshot with Color Highlighter enabled (Gutter Color disabled):
![test css_color-highlighter](https://cloud.githubusercontent.com/assets/1308190/4005489/a5576e66-2992-11e4-8590-1bfcd01cc077.png)

I see that Color Highlighter show different tone of blue and green color for the lines named blue and green, but Gutter Color show others color very far of the blue and Green.
Why is great difference, both use ImageMagick.

**Comment by jbrooksuk** [comment]
@tdhsmith hey! I guess this is definitely not happening?

**Comment by LOQUILLO** [comment]
Again, not fixed:
![gutter-color-bug](https://cloud.githubusercontent.com/assets/1308190/9506458/dd2dcf4e-4c0d-11e5-9755-e60257009a20.png)

**Comment by tdhsmith** [comment]
Weirdly enough, a change *just* landed in the ImageMagick source this past week, prompted by my original bug report.

http://www.imagemagick.org/script/changelog.php
> 2015-08-15 6.9.2-1 Cristy <quetzlzacatenango@image...>
> * Support color compliance for TXT format (e.g. -define txt:compliance=css) (bug report from Anthony).
> * **The first argument of a hsl() color is integer, not percentage (reference http://www.imagemagick.org/discourse-server/viewtopic.php?f=2&t=26072).**
> * Limit -fx recursive to avoid stack overflow (reference http://www.imagemagick.org/discourse-server/viewtopic.php?f=3&t=28200).
> * Support inline images with data URLS.

But that *only* fixes their HSL/HSLA to be CSS standards compliant. So something like `hsl(0.7, 0.1, 0.9)` or anything other than `hsl(<angle>,<percent>,<percent>)`, won't work.

Now CSS is pretty assuredly the number 1 use-case for GutterColor, but when the problem was originally presented, it seemed like people were interested in supporting arbitrary color formats. So the question is do we call this solved, or still look into parsing out and converting colors so we can support everything?

**Comment by LOQUILLO** [comment]
Then the problem is ImageMagick, or installing some other old IM versión maybe solve this.

**Comment by tdhsmith** [comment]
Yes the problem has always been ImageMagick. It does not have the parsing to be able to identify the position of a percentage sign or the difference between `(10, 100%)` and `(10%, 100%)`. ColorHighlighter solved it some time ago by [normalizing all numbers before sending to ImageMagick](https://github.com/Monnoroch/ColorHighlighter/blob/f416f6fab26b50566b134ae8f4b176af21e12b48/ColorHighlighter.py#L401).

**Comment by LOQUILLO** [comment]
Hi @tdhsmith then which versión of IM you recommend?

**Comment by jbrooksuk** [comment]
@tdhsmith we should probably implement normalization of numbers too.

**Comment by LOQUILLO** [comment]
Now, it's much better but transparency isn't working:
![gutter_color_test](https://cloud.githubusercontent.com/assets/1308190/9564116/1412bc40-4e61-11e5-8408-47f5ab026ac2.png)
